src/bin/nydusd/main.rs

```diff
@@ -404,6 +404,16 @@ fn prepare_commandline_options() -> App<'static, 'static> {
                 .required(false)
                 .global(true),
         )
+        .arg(
+            Arg::with_name("log-rotate-size")
+                .long("log-rotate-size")
+                .short("r")
```

Comment:
As log rotation is not always needed, we'd better not give it a shore argument `-r`. It can reserved for other potential arguments.